blocks/footer/__container/footer__container.css

```diff
@@ -0,0 +1,27 @@
+.footer__container{
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+}
+
+@media screen and (max-width: 1199px) {
+  .footer__container {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+@media screen and (max-width: 659px) {
+  .footer__container {
+    width: 320px;
+    padding-left: 74px;
```

Comment:
As advised, I have moved the padding assignment for mobile devices to the element with the class name footer to centralize the styling. Furthermore, I have updated the padding notation to use shorthand padding: 40px 20px; for improved clarity. These modifications enhance code consistency and readability while adhering to best practices for styling.

Reply:
@ElenaPodv Let's ensure that our paddings are styled and reassigned only in one place. The mobile layout doesn't align with the design because right now we're adjusting paddings __for two different elements__:

_1. Currently, we have paddings assigned for tablet screens (they also cascade to mobile) for the element with the class `footer`_

![Screenshot 2024-01-03 at 10 53 26 AM](https://github.com/p-foundation/pink-app/assets/76534205/368de713-c3f3-4cbe-ac8f-05685953ca16)

_2. At the same time, we have paddings assigned for mobile devices for the element with the class `footer__container`_

![Screenshot 2024-01-03 at 10 56 40 AM](https://github.com/p-foundation/pink-app/assets/76534205/702a70df-ed8e-42da-91eb-1d084a9fefe9)

**To fix this, let's consolidate our paddings on the element with the class `footer`:**
1. Please remove `padding: 40px 74px;` from the element with the class `footer__container`
2. Add the following media query for mobile devices to the `footer` element:
```CSS
@media screen and (max-width: 659px) {
  .footer {
    padding: 40px 0;
  }
}
```

Reply:
Done